meilisearch-http/src/routes/document.rs

```diff
@@ -149,7 +181,8 @@ async fn update_multiple_documents(
     let index = data
         .db
         .open_index(&path.index_uid)
-        .ok_or(Error::index_not_found(&path.index_uid))?;
+        .ok_or(Error::index_not_found(&path.index_uid))
+        .or(create_index(&data, &path.index_uid))?;
```

Comment:
This statement will be called always before every thing related to this line, you should use `or_else` instead.